plugins/modules/cmci_create.py

```diff
@@ -79,29 +84,29 @@
   type: str
 cpsm_reason:
   description:
-    - The character value of the REASON code returned by each CICSPlex SM API command.
-      For a list of REASON character values, see
+    - The character value of the REASON code returned by each CICSPlex SM API
+      command. For a list of REASON character values, see
       https://www.ibm.com/support/knowledgecenter/SSGMCP_5.6.0/reference-system-programming/commands-cpsm/eyup2ky.html.
```

Comment:
Can we convert this to a link?

Reply:
This is in the RETURN block instead of DOCUMENTATION, so a URL will be detected and generated as a link by default. Formatting tags won't work here though. @stewartfrancis